packages/playback-core/src/request-errors.ts

```diff
@@ -0,0 +1,255 @@
+      tokenNamePrefix,
+    });
+    const context = i18n(`Compact JWT string: {token}`, translate).format({
+      token,
+    });
+    const mediaError = new MediaError(message, mediaErrorCode, true, context);
+    mediaError.errorCategory = category;
+    mediaError.muxCode = MuxErrorCode.NETWORK_TOKEN_MALFORMED;
+    mediaError.data = resp;
+    return mediaError;
+  }
+
+  if (status >= 500) {
+    /**
+     * @TODO We plausibly should have some basic retry logic for all other 500 status
+     * cases (CJP)
+     **/
+    const mediaError = new MediaError('', mediaErrorCode, true);
+    mediaError.errorCategory = category;
+    mediaError.muxCode = MuxErrorCode.NETWORK_UNKNOWN_ERROR;
+    /** @TODO Add error msg + context crud here (NOT YET DEFINED) (CJP) */
+    return mediaError;
+  }
+
+  if (status === 403) {
+    if (jwtObj) {
+      if (isJWTExpired(jwtObj, requestTime)) {
+        const dateOptions: any = {
+          timeStyle: 'medium',
+          dateStyle: 'medium',
+        };
+        // E.g. for DRM: "The video’s secured drm-token has expired."
+        // E.g. for Video: "The video’s secured playback-token has expired."
+        const message = i18n(`The video’s secured {tokenNamePrefix}-token has expired.`, translate).format({
+          tokenNamePrefix,
+        });
+        const context = i18n(`Expired at: {expiredDate}. Current time: {currentDate}.`, translate).format({
+          expiredDate: new Intl.DateTimeFormat('en', dateOptions).format(jwtObj.exp ?? 0 * 1000),
+          currentDate: new Intl.DateTimeFormat('en', dateOptions).format(requestTime),
+        });
+        const mediaError = new MediaError(message, mediaErrorCode, true, context);
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.NETWORK_TOKEN_EXPIRED;
+        mediaError.data = resp;
+        return mediaError;
+      }
+      if (isJWTSubMismatch(jwtObj, playbackId)) {
+        const message = i18n(
+          `The video’s playback ID does not match the one encoded in the {tokenNamePrefix}-token.`,
+          translate
+        ).format({
+          tokenNamePrefix,
+        });
+        const context = i18n(
+          `Specified playback ID: {playbackId} and the playback ID encoded in the {tokenNamePrefix}-token: {tokenPlaybackId}`,
+          translate
+        ).format({
+          tokenNamePrefix,
+          playbackId,
+          tokenPlaybackId: jwtObj.sub,
+        });
+        const mediaError = new MediaError(message, mediaErrorCode, true, context);
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.NETWORK_TOKEN_SUB_MISMATCH;
+        mediaError.data = resp;
+        return mediaError;
+      }
+      if (isJWTAudMissing(jwtObj, expectedAud)) {
+        const message = i18n(`The {tokenNamePrefix}-token is formatted with incorrect information.`, translate).format({
```

Comment:
Nit: does it make sense to add more specificity here? (Relying on `isJWTAudMissing` to infer the `aud` claim is at issue here.)

Strawman message (rough wording): `The {tokenNamePrefix}-token has a missing or incorrect audience claim.`

Reply:
Ah okay. I see that detail is in the error context.